src/components/tab-bar/tab-bar.scss

```diff
@@ -14,6 +14,8 @@ $tab-scroller-fade-width: 65;
 
 :host {
     display: block;
+    position: relative;
+    z-index: 3;
```

Comment:
Do we have to use `z-index` here? I would prefer if we could do without it

Reply:
![image](https://user-images.githubusercontent.com/35954987/84884818-7c93db80-b092-11ea-921a-cb0aea768797.png)
Yes, this is because hovered list items (as well as other things which may have a z-index) will render on top of the tab-bar. And this is because we have an absolute position on `.tab-content`. But now since that's removed, this is not needed anymore. So I'll revet this.